website/docs/d/is_reservations.html.markdown

```diff
@@ -0,0 +1,151 @@
+---
+subcategory: "VPC infrastructure"
+layout: "ibm"
+page_title: "IBM : Reservations"
+description: |-
+  Manages IBM Cloud infrastructure reservations.
+---
+
+# ibm_is_reservations
+Retrieve information of an existing reservations as a read only data source.
```

Comment:
link here please

Reply:
There is no link yet, I will update once links are available